components/button/ConfirmBtn.tsx

```diff
@@ -1,9 +1,12 @@
+import { ButtonProps } from "./BackBtn";
```

Comment:
Good job reusing the typing here.
You could think about an isolated `button/types.ts` file.